Assets/ZSSRichTextEditor.js

```diff
@@ -107,7 +107,12 @@ ZSSEditor.init = function(callbacker, logger) {
         // Ensure we only insert plaintext from the pasteboard
         e.preventDefault();
         var plainText = (e.originalEvent || e).clipboardData.getData('text/plain');
-        document.execCommand('insertText', false, plainText);
+        if (plainText.length > 0) {
```

Comment:
It's ok for now to keep as it is, depending on the progress you make, we can come back here and handle the other scenarios.